test.js

```diff
@@ -979,4 +979,30 @@ describe('Express Mongo Sanitize', function () {
         );
     });
   });
+
+  describe('sanitize', function () {
+    it('should not throw a NPE when options is not passed into the function', function () {
+      const req = {
+        body: {
+          hello: 'world',
+        },
+      };
+
+      const resp = sanitize.sanitize(req);
```

Comment:
Can we change this to 

```suggestion
      const resp = sanitize.sanitize(req.body);
```

To reflect the fact that the caller should be sanitising req.body instead of the entire request object?